watertap/property_models/unit_specific/activated_sludge/modified_asm2d_properties.py

```diff
@@ -512,4 +675,28 @@ def calculate_scaling_factors(self):
 
         if self.is_property_constructed("TSS"):
             if iscale.get_scaling_factor(self.TSS) is None:
-                iscale.set_scaling_factor(self.TSS, 1)
+                iscale.set_scaling_factor(self.TSS, 1e1)
+
+        if self.is_property_constructed("COD"):
+            if iscale.get_scaling_factor(self.COD) is None:
+                iscale.set_scaling_factor(self.COD, 1e1)
+
+        if self.is_property_constructed("BOD5"):
+            if iscale.get_scaling_factor(self.BOD5) is None:
+                iscale.set_scaling_factor(self.BOD5, 1e1)
+
+        if self.is_property_constructed("SNKj"):
+            if iscale.get_scaling_factor(self.SNKj) is None:
+                iscale.set_scaling_factor(self.SNKj, 1e2)
+
+        if self.is_property_constructed("SNOX"):
+            if iscale.get_scaling_factor(self.SNOX) is None:
+                iscale.set_scaling_factor(self.SNOX, 1e3)
+
+        if self.is_property_constructed("SP_organic"):
+            if iscale.get_scaling_factor(self.SP_organic) is None:
+                iscale.set_scaling_factor(self.SP_organic, 1e2)
+
+        if self.is_property_constructed("SP_inorganic"):
+            if iscale.get_scaling_factor(self.SP_inorganic) is None:
+                iscale.set_scaling_factor(self.SP_inorganic, 1e3)
```

Comment:
Do we need to scale the expressions? Have you tried without this added scaling?

Reply:
Just tried removing scaling in the latest commit - it still fails